service/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is the purpose of this file, is there any Arquillian test? Keep all Arquillian-related artifacts in a separate commit in a separate branch in case we need them.